### PR TITLE
Add ABOUT_ME.md file

### PR DESCRIPTION
**Pull request by kumarprakash1636**
If applied, this commit will add ABOUT_ME.md file.

### CODE REVIEW

File: ABOUT_ME.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INTRODUCTION:-
+
+Hello everyone, my name is Prakash Kumar.I am pursuing B.Tech from Silicon Institute of Technology.
+Currently, I am in 8th semester and doing intern at Surya Digital.My hobbies are listening songs and going to gym.
+
```
Comment (sudarshan-dodiya) [inline ABOUT_ME.md]: This does not seem like proper Markdown formatting, please make sure you use Markdown syntax to write in a markdown file.

File: ABOUT_ME.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+INTRODUCTION:-
+
+Hello everyone, my name is Prakash Kumar.I am pursuing B.Tech from Silicon Institute of Technology.
+Currently, I am in 8th semester and doing intern at Surya Digital.My hobbies are listening songs and going to gym.
```
Comment (sudarshan-dodiya) [inline ABOUT_ME.md]:
```suggestion
Currently, I am in 8th semester and doing internship at Surya Digital. My hobbies are listening songs and going to gym.
```
The grammar of this sentence does not seem right to me, please use a tool like Grammarly to proofread your sentences.

**Review by sudarshan-dodiya — CHANGES_REQUESTED** [review]
The title and the commit message does not follow the rules of a good commit message: https://coda.io/d/_dHOXTAtFBHO/Creating-Pull-Requests_sulg3#_luNjv

File: ABOUT_ME.md (hunk 2 above)
Comment (kumarprakash1636) [reply ABOUT_ME.md]: Thanks for your feedback.

File: ABOUT_ME.md (hunk 1 above)
Comment (sudarshan-dodiya) [reply ABOUT_ME.md]: It's still not proper Markdown formatting. Please make sure that I don't have to point out same thing again.

File: ABOUT_ME.md (hunk 2 above)
Comment (sudarshan-dodiya) [reply ABOUT_ME.md]: It's more of a practice to instead mention if you have addressed the feedback or not. It might be the case that you are not addressing it the way I want and have implemented something in a much better way, at that time having proper context helps.

**Comment by sudarshan-dodiya** [comment]
You still haven't updated the PR Title, and have not followed the instructions on writing a good commit message.

File: ABOUT_ME.md (hunk 2 above)
Comment (kumarprakash1636) [reply ABOUT_ME.md]: Understood.

File: .idea/vcs.xml
Comment (sudarshan-dodiya) [inline .idea/vcs.xml:1]: Why is this file here? Please make sure to gitignore this.

**Review by sudarshan-dodiya — CHANGES_REQUESTED** [review]

File: .idea/vcs.xml
Comment (kumarprakash1636) [reply .idea/vcs.xml:1]: Removed

**Review by sudarshan-dodiya — APPROVED** [review]
